### PR TITLE
Mage Tower 4F -> 5F

### PR DESCRIPTION
**Pull request by vtange**
Leaving the new Gabija lines since they are 20151001.

**to IMCgames**

ETC Line 5616 - I tried shortening this to fit the UI but I'm sure it's still too long. Can you fix the UI?

### CODE REVIEW

**Comment by terrapin47** [comment]
:+1:

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -4945,43 +4945,43 @@ QUEST_LV_0100_20150317_004944	Check the immobile Mineloader
 QUEST_LV_0100_20150317_004945	There is a Mineloader at the Mage Tower that act as a guard. Check the immobile Mineloader.
 QUEST_LV_0100_20150317_004946	Defeat the Mineloader that is attacking
 QUEST_LV_0100_20150317_004947	Defeat the Mineloader that started to move suddenly
-QUEST_LV_0100_20150317_004948	Grita is waiting at the fourth floor of the Mage Tower. Talk to Grita.
+QUEST_LV_0100_20150317_004948	$Grita is waiting at the fourth floor of the Mage Tower. Go to Grita.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:4952]: at the fourth floor 

or

on the fourth floor?

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -4945,43 +4945,43 @@ QUEST_LV_0100_20150317_004944	Check the immobile Mineloader
...
+QUEST_LV_0100_20150317_004980	$Helgasercle created Magic Suppressors on the 5th floor to weaken the Mage Tower's power. Destroy the 3rd Suppressor located at the Small Hall.
 QUEST_LV_0100_20150317_004981	$Destroy Helgasercle's 4th Magic Suppressor
-QUEST_LV_0100_20150317_004982	$Helgasercle created magic suppressors on the 5th floor to weaken the Mage Tower's power. Destroy the 4th suppressor at the Reading Room.
+QUEST_LV_0100_20150317_004982	$Helgasercle created Magic Suppressors on the 5th floor to weaken the Mage Tower's power. Destroy the 4th Suppressor at the Reading Room.
 QUEST_LV_0100_20150317_004983	$Find Helgasercle
-QUEST_LV_0100_20150317_004984	You destroyed all four magic suppressors that Helgasercle prepared. It is now time to liberate the Mage Tower from the Demon's power. Find Helgasercle at the Great Hall.
+QUEST_LV_0100_20150317_004984	You destroyed all four Magic Suppressors that Helgasercle prepared. It is now time to liberate the Mage Tower from the Demon's power. Find Helgasercle at the Great Hall.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Demon's -> demon's

File: QUEST_LV_0100.tsv (hunk 1 above)
Comment (terrapin47) [inline QUEST_LV_0100.tsv:4952]: both work

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**